### ISSUE TITLE
Multiple source planes

### ISSUE THREAD

**Issue by smartalecH** [facebookresearch/fmmax#24]
It'd be great to be able to specify multiple source planes within a simulation. 

Similarly, it'd be great to have an API for *volumetric* sources (e.g. imagine if the source had a finite thickness associated with it).

**Comment by mfschubert** [comment]
Yes, we could update one of the examples to include multiple source planes. I anticipate the code changes will be entirely in the usage of the fmmax API, and that no internal changes will be required.
